geomesa-process/geomesa-process-vector/src/test/scala/org/locationtech/geomesa/process/transform/ArrowConversionProcessTest.scala

```diff
@@ -58,6 +60,12 @@ class ArrowConversionProcessTest extends Specification {
       }
     }
 
+    "encode a generic empty feature collection with dictionary values without leaking memory" in {
+      // This returns an empty iterator.
+      process.execute(new ListFeatureCollection(sft), null, null, null, Seq("name"), null, null, null, null, null)
+      ArrowAllocator.getAllocatedMemory(sft.getTypeName) must equalTo(0)
```

Comment:
we should make the type name more unique to avoid interference from any other tests